### PR TITLE
BZ1979811: Clarify Ansible scope

### PR DESCRIPTION
**Pull request by stoobie**
Bug fix

This PR addresses https://bugzilla.redhat.com/show_bug.cgi?id=1979811)

Changes proposed in this pull request:
- Changed verbiage to indicate that the version of Ansible included with oVirt/RHV is limited, not the full product.

Preview: https://ovirt.github.io/ovirt-site/previews/2637/documentation/administration_guide/index.html#chap-Automating_RHV_Configuration_using_Ansible

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @stoobie 
This pull request needs review by: @michalskrivanek

### CODE REVIEW

**Comment by janosdebugs** [comment]
Your preview URL is https://ovirt.github.io/ovirt-site/previews/2637

File: source/documentation/administration_guide/chap-Automating_RHV_Configuration_using_Ansible.adoc [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
 Ansible Tower is a graphically enabled framework accessible through a web interface and REST APIs for Ansible. If you want support for Ansible Tower, then you must have an Ansible Tower license, which is not part of the {virt-product-fullname} subscription.
 ====
-Ansible is shipped with {virt-product-fullname}. To install Ansible, run the following command on the {engine-name} machine:
-
-[options="nowrap" subs="normal"]
-----
-# dnf install ansible
-----
 
 See the link:http://docs.ansible.com/[Ansible Documentation] for alternate installation instructions, and information about using Ansible.
```
Comment (RichardHoch) [inline source/documentation/administration_guide/chap-Automating_RHV_Configuration_using_Ansible.adoc:13]: suggestion: lower-case d in "documentation."

**Comment by RichardHoch** [comment]
@stoobie: One small comment; otherwise, LGTM!

**Review by RichardHoch — APPROVED** [review]

**Comment by janosdebugs** [comment]
@sandrobonazzola could you please change the branch protection rules to require "Build Preview" and "Content Check", but not "build" please?

**Pull request merged**